### ISSUE TITLE
Crash when invoking .destroy()

### ISSUE THREAD

**Issue by AVVS** [gre/diaporama#38]
Stack trace:

```
Error in event handler for (unknown): TypeError: undefined is not a function
    at Object.StoreTransitions.destroy (chrome-extension://nebbhkdcphlabifnafeopppcnjmnjoef/lib.min.js:17457:20)
    at Object.DiaporamaRenderingCanvas.destroy (chrome-extension://nebbhkdcphlabifnafeopppcnjmnjoef/lib.min.js:17514:22)
    at Object.assign.destroy (chrome-extension://nebbhkdcphlabifnafeopppcnjmnjoef/lib.min.js:16813:21)
    at React.createClass.componentWillUnmount (chrome-extension://nebbhkdcphlabifnafeopppcnjmnjoef/lib.min.js:16661:20)
    at ReactCompositeComponentMixin.unmountComponent (chrome-extension://nebbhkdcphlabifnafeopppcnjmnjoef/lib.min.js:30097:14)
    at Object.ReactReconciler.unmountComponent (chrome-extension://nebbhkdcphlabifnafeopppcnjmnjoef/lib.min.js:38006:22)
    at ReactCompositeComponentMixin.unmountComponent (chrome-extension://nebbhkdcphlabifnafeopppcnjmnjoef/lib.min.js:30103:21)
    at Object.ReactReconciler.unmountComponent (chrome-extension://nebbhkdcphlabifnafeopppcnjmnjoef/lib.min.js:38006:22)
    at Object.ReactChildReconciler.unmountChildren (chrome-extension://nebbhkdcphlabifnafeopppcnjmnjoef/lib.min.js:28429:23)
    at ReactDOMComponent.ReactMultiChild.Mixin.unmountChildren (chrome-extension://nebbhkdcphlabifnafeopppcnjmnjoef/lib.min.js:36745:28)
```

```js
StoreTransitions.prototype = {
  destroy: function () {
    for (var t in this.ts) {
      this.ts[t].t.destroy();    // crash happens here      <---------------- 
    }
    this.ts = null;
  },
```

**Comment by AVVS** [comment]
Version .15 was fine, when updated to .18 this started happening

**Comment by AVVS** [comment]
Checked .17 - no issue there as well, so it must be the latest release

**Comment by gre** [comment]
Sorry for that, should be fixed in `1.1.19`
